src/MySqlConnector/Core/SchemaProvider.cs

```diff
@@ -58,6 +60,54 @@ public DataTable GetSchema(string collectionName)
 			return dataTable;
 		}
 
+		private void FillDataSourceInformation(DataTable dataTable)
+		{
+			dataTable.Columns.AddRange(new [] {
+				new DataColumn("CompositeIdentifierSeparatorPattern", typeof(string)),
+				new DataColumn("DataSourceProductName", typeof(string)),
+				new DataColumn("DataSourceProductVersion", typeof(string)),
+				new DataColumn("DataSourceProductVersionNormalized", typeof(string)),
+				new DataColumn("GroupByBehavior", typeof(GroupByBehavior)),
+				new DataColumn("IdentifierPattern", typeof(string)),
+				new DataColumn("IdentifierCase", typeof(IdentifierCase)),
+				new DataColumn("OrderByColumnsInSelect", typeof(bool)),
+				new DataColumn("ParameterMarkerFormat", typeof(string)),
+				new DataColumn("ParameterMarkerPattern", typeof(string)),
+				new DataColumn("ParameterNameMaxLength", typeof(int)),
+				new DataColumn("QuotedIdentifierPattern", typeof(string)),
+				new DataColumn("QuotedIdentifierCase", typeof(IdentifierCase)),
+				new DataColumn("ParameterNamePattern", typeof(string)),
+				new DataColumn("StatementSeparatorPattern", typeof(string)),
+				new DataColumn("StringLiteralPattern", typeof(string)),
+				new DataColumn("SupportedJoinOperators", typeof(SupportedJoinOperators))
+			});
+
+			var row = dataTable.NewRow();
+			row["CompositeIdentifierSeparatorPattern"] = @"\.";
+			row["DataSourceProductName"] = "MysqlConnector";
```

Comment:
`DataSourceProductName` is "The name of the product accessed by the provider, such as 'Oracle' or 'SQLServer'." (https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/common-schema-collections)

I don't know if we can accurately determine the remote database (MySQL, Percona, MariaDB, Amazon, etc.) so it might be best to just return `"MySQL"`.